examples/examples_dotnet_test.go

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 // +build dotnet all
 
-package examples
```

Comment:
This should stay as examples

Reply:
Something I failed to revert after running into some go mod issues. Fixed now across the examples.